core/modules/system/system.module

```diff
@@ -2824,22 +2824,44 @@ function system_get_info($type, $name = NULL) {
  *   An associative array of module information.
  */
 function _system_rebuild_module_data() {
+  require_once BACKDROP_ROOT . '/core/includes/install.inc';
+
   // Find modules
   $modules = backdrop_system_listing('/^' . BACKDROP_PHP_FUNCTION_PATTERN . '\.module$/', 'modules', 'name', 0);
 
-  // Include the installation profile in modules that are loaded.
+  // Include the installation profile(s) in modules that are loaded.
+  // This will include the current profile, then loop through any base profiles
+  // as well.
```

Comment:
```suggestion
  // Include the installation profile(s) in modules that are loaded. This will
  // include the current profile, then loop through any base profiles as well.
```